### PR TITLE
fix #33578 cikavosti.com

### LINKED ISSUE #33578: cikavosti.com
### Issue URL (Ads)
[https://cikavosti.com/vcheni-z-yasuvali-u-yakomu-viczi-lyudi-chastishe-zajmayutsya-seksom/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fcikavosti.com%2Fvcheni-z-yasuvali-u-yakomu-viczi-lyudi-chastishe-zajmayutsya-seksom%2F)
### Comment
Username: @big-u-boss
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.0.14
Browser: | Chrome
Stealth mode options: | Hide your search queries, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Hide your Referrer from third-parties
Filters: | AdGuard Russian,<br>AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by tvinzz**
https://github.com/AdguardTeam/AdguardFilters/issues/33578

### CODE REVIEW

File: RussianFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -217,6 +217,9 @@ list-org.com#$?#.menu > div[class="dmenu sticky"]:has(> center > ins.adsbygoogle
 reporter-ua.com#?#.row > .sidebar > .block:has(> h4:contains(mgid))
 4mama.ua#?#.sidebar a.sb-projects-header-lbl:contains(Новости партнеров)
 mysitem.ru#?#.custom-none:has(> div[id^="yandex_rtb"])
+cikavosti.com#?#aside > .widget:has(noindex > a[target="_blank"][rel="nofollow"])
+cikavosti.com#?#aside > .widget:has(> .widget-top:contains(Реклама))
+cikavosti.com#?#.a-single > div[id]:has(a[target="_blank"][rel="nofollow"][href^="//goods.redtram.com"])
```
Comment (Alex-302) [inline RussianFilter/sections/css_extended.txt]: `[target="_blank"][rel="nofollow"]` is redundant

File: RussianFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -217,6 +217,9 @@ list-org.com#$?#.menu > div[class="dmenu sticky"]:has(> center > ins.adsbygoogle
 reporter-ua.com#?#.row > .sidebar > .block:has(> h4:contains(mgid))
 4mama.ua#?#.sidebar a.sb-projects-header-lbl:contains(Новости партнеров)
 mysitem.ru#?#.custom-none:has(> div[id^="yandex_rtb"])
+cikavosti.com#?#aside > .widget:has(noindex > a[target="_blank"][rel="nofollow"])
```
Comment (Alex-302) [inline RussianFilter/sections/css_extended.txt]: don't block the links to the blogs. And it is not an ad or annoyance

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/css_extended.txt (hunk 1 above)
Comment (tvinzz) [reply RussianFilter/sections/css_extended.txt]: @Alex-302,
deleted redundant part of the rule

Comment (Alex-302) [reply RussianFilter/sections/css_extended.txt]: I mean this part `:has(a[target="_blank"][rel="nofollow"][href^="//goods.redtram.com"])`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: RussianFilter/sections/css_extended.txt (hunk 1 above)
Comment (tvinzz) [reply RussianFilter/sections/css_extended.txt]: fixed

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**